ambry-store/src/test/java/com.github.ambry.store/CompactionPolicyTest.java

```diff
@@ -22,13 +22,13 @@
 import com.github.ambry.utils.Time;
 import com.github.ambry.utils.Utils;
 import com.github.ambry.utils.UtilsTest;
-import java.util.ArrayList;
```

Comment:
these removals seem to be causing build errors.
https://travis-ci.org/linkedin/ambry/builds/240916956